etc/fixtures/initial-seed/analyses.json

```diff
@@ -701,19 +701,9 @@
         "username": "user06"
       },
       {
-        "event": "ready",
```

Comment:
Instead of modifying an existing analysis in the fixtures and change it, lets create a new analysis in the fixtures that is a newly loaded case instead. You can choose any of the existing cases that we upload to rosalution to streamline this.

After a new analysis is added to this fixture, this case can get its history reverted back to the original where it had a decline state.